### PR TITLE
Emml1194

### PR DESCRIPTION
**Pull request by MarcinKrawczuk**
I am sending the catalog description, I have a pdf copy of the ms. if needed.
[EMML 1194 opis.pdf](https://github.com/BetaMasaheft/Manuscripts/files/7650151/EMML.1194.opis.pdf)

### CODE REVIEW

File: EMML/1001-2000/EMML1194.xml [hunk 1]
```diff
@@ -36,46 +36,232 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
...
+
+<additions>
+
+<list>
+
+<item xml:id="a1">
+<locus target="#306"/>
+<desc type="ScribalSignature">The name of the scribe is <persName role="scribe" ref="PRS13645Qalama">Qalama Warq Lawtē</persName>. According to the catalog, pages 1-95
+have been copied by <persName role="scribe" ref="PRS13646Makwannen">Makʷannen Gabra Śellāsē</persName>.</desc>
+<q xml:lang="gez">ቃለመ፡ ወርቅ፡ ለውጤ፡ ዘአሸማ፡ ቂርቆስ፡ ወዘጓ፡ ጊ<supplied reason="omitted">ዮርጊስ</supplied>፡</q>
```
Comment (thea-m) [inline EMML/1001-2000/EMML1194.xml:234]: ቃለመ or ቀለመ?

File: EMML/1001-2000/EMML1194.xml [hunk 2]
```diff
@@ -36,46 +36,232 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
...
+                   </handNote>
                   </handDesc>
+
+<additions>
+
+<list>
+
+<item xml:id="a1">
+<locus target="#306"/>
+<desc type="ScribalSignature">The name of the scribe is <persName role="scribe" ref="PRS13645Qalama">Qalama Warq Lawtē</persName>. According to the catalog, pages 1-95
```
Comment (thea-m) [inline EMML/1001-2000/EMML1194.xml:232]: Lawwəṭṭe, I think?

File: EMML/1001-2000/EMML1194.xml [hunk 3]
```diff
@@ -36,46 +36,232 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
...
+have been copied by <persName role="scribe" ref="PRS13646Makwannen">Makʷannen Gabra Śellāsē</persName>.</desc>
```
Comment (thea-m) [inline EMML/1001-2000/EMML1194.xml:233]: Śəllāse

**Review by thea-m — CHANGES_REQUESTED** [review]

File: EMML/1001-2000/EMML1194.xml (hunk 1 above)
Comment (MarcinKrawczuk) [reply EMML/1001-2000/EMML1194.xml:234]: ቀ

File: EMML/1001-2000/EMML1194.xml (hunk 3 above)
Comment (MarcinKrawczuk) [reply EMML/1001-2000/EMML1194.xml:233]: Of course :/

**Review by thea-m — APPROVED** [review]
Thank you!

**Pull request merged**